tests/test_routes.py

```diff
@@ -479,7 +479,7 @@ def test_get_order_list_by_name(self):
         orders = self._create_orders(3)
         test_name = orders[0].customer_name
         name_orders = [order for order in orders if order.customer_name == test_name]
-        resp = self.client.get(BASE_URL, query_string=f"name={test_name}")
+        resp = self.client.get(BASE_URL, query_string=f"customer_name={test_name}")
```

Comment:
can we have a similar test for customer_name for empty customer_name?

Reply:
Actually, we already have "test_get_order_by_name()" and "test_get_order_by_name_empty()", I am not sure whether we still need "test_get_order_list_by_name()" here. 
I didn’t delete it because it wasn’t written by me; I only modified the query_string within it.

Reply:
> can we have a similar test for customer_name for empty customer_name?

Actually, we already have "test_get_order_by_name()" and "test_get_order_by_name_empty()", I am not sure whether we still need "test_get_order_list_by_name()" here.
I didn’t delete it because it wasn’t written by me; I only modified the query_string within it.

Reply:
ah okay. let's just keep it.